runtime/src/impls.rs

```diff
@@ -189,11 +189,12 @@ mod multiplier_tests {
             let mut fm = Multiplier::one();
             // See the example in the doc of `TargetedFeeAdjustment`. are at least 0.234, hence
             // `fm > 1.234`.
+            // BlockTime increased from 3 to 12s ==> decreaced to 0.050
             for _ in 0..DAYS {
                 let next = runtime_multiplier_update(fm);
                 fm = next;
             }
-            assert!(fm > Multiplier::saturating_from_rational(1234, 1000));
+            assert!(fm > Multiplier::saturating_from_rational(1050, 1000));
```

Comment:
@simonsso What is fm? and How does this affect the block time?

Reply:
This is for the unit test which fails.

Reply:
https://github.com/paritytech/substrate/blob/4c3a55e7ca5c4c85c1eb53fd82ed71029d952510/frame/transaction-payment/src/lib.rs 

Should be 17220 * 0.8125 * 0.00001 = 0.14040 => 1140

Reply:
The theoretical value does however not work in the testcase and I did not want to dig any deeper